api/match.py

```
@@ -0,0 +1,109 @@
+import requests
+from utils.db import get_stadium_by_team_name, get_team_id_by_name, get_match_id_by_teams_and_date
+
+def get_match_info_by_date(date: str):
+    url = f"https://api-gw.sports.naver.com/schedule/games?fields=basic%2Cschedule%2Cbaseball&upperCategoryId=kbaseball&fromDate={date}&toDate={date}&size=500"
```

Pylint (3.3.7)

[convention] Line too long (159/100)

(C0301)